src/main/resources/assets/computercraft/lua/rom/apis/rednet

```diff
@@ -8,8 +8,16 @@ local tHostnames = {}
 
 function open( sModem )
 	if type( sModem ) ~= "string" then
-		error( "expected string", 2 )
+	    for key,value in ipairs({"left","right","up","bottom","back","front"}) do
```

Comment:
You seem to be using a weird mix of spaces and tabs here. The rest of this function seems to be using tabs, so you might want to change this block to use tabs.

I'd also recommend using `rs.getSides()` instead of manually enumerating them. Perhaps even use `peripheral.getName()`.

You might also want to use more descriptive variable names than `key` and `value`: something like `n, sSide`  would be OK.